modules/statistics/statistics.js

```diff
@@ -283,6 +290,63 @@ Statistics.prototype.removeByteSentStatsListener = function(listener) {
         listener);
 };
 
+/**
+ * Add a listener that would be notified on a LONG_TASKS_STATS event.
+ *
+ * @param {Function} listener a function that would be called when notified.
+ * @returns {void}
+ */
+Statistics.prototype.addPerformanceStatsListener = function(listener) {
+    this.eventEmitter.on(StatisticsEvents.LONG_TASKS_STATS, listener);
+};
+
+/**
+ * Creates an instance of {@link PerformanceObserverStats} and starts the
+ * observer that records the stats periodically.
+ *
+ * @returns {void}
+ */
+Statistics.prototype.attachPerformanceStats = function(conference) {
+    if (!browser.supportsPerformanceObserver()) {
+        logger.warn('Performance observer for long tasks not supported by browser!');
+
+        return;
+    }
+
+    this.performanceObserverStats = new PerformanceObserverStats(
+        this.eventEmitter,
+        Statistics.performanceStatsInterval);
+
+    conference.on(
+        JitsiConferenceEvents.CONFERENCE_JOINED,
+        () => this.performanceObserverStats.startObserver());
+    conference.on(
+        JitsiConferenceEvents.CONFERENCE_LEFT,
+        () => this.performanceObserverStats.stopObserver());
+};
+
+/**
+ * Obtains the current value of the performance statistics.
+ *
+ * @returns {Object|null} stats object if the observer has been
+ * created, null otherwise.
+ */
+Statistics.prototype.getPerformanceStats = function() {
```

Comment:
ah I see you actually wrote a getter here, but it's not used in `JitsiConference`.

Reply:
I will switch to using this getter in `JitsiConference` instead of calling on the `performanceObserverStats` directly.